prime/src/data/BalanceSheet.ts

```diff
@@ -3,53 +3,28 @@ import { GN } from 'shared/lib/data/GoodNumber';
 
 import { getAmountsForLiquidity, getValueOfLiquidity } from '../util/Uniswap';
 import { UniswapPosition } from './actions/Actions';
-import {
-  ALOE_II_LIQUIDATION_INCENTIVE,
-  ALOE_II_MAX_LEVERAGE,
-  ALOE_II_SIGMA_MAX,
-  ALOE_II_SIGMA_MIN,
-} from './constants/Values';
+import { ALOE_II_LIQUIDATION_INCENTIVE, ALOE_II_MAX_LEVERAGE } from './constants/Values';
 import { Assets, Liabilities, LiquidationThresholds } from './MarginAccount';
 
 const MIN_SQRT_RATIO = GN.fromJSBI(TickMath.MIN_SQRT_RATIO, 96, 2);
 const MAX_SQRT_RATIO = GN.fromJSBI(TickMath.MAX_SQRT_RATIO, 96, 2);
-const ONE = GN.one(18);
 
-function _computeProbePrices(sqrtMeanPriceX96: GN, sigma: GN, nSigma: number): [GN, GN] {
-  if (sigma.lt(ALOE_II_SIGMA_MIN)) sigma = ALOE_II_SIGMA_MIN;
-  else if (sigma.gt(ALOE_II_SIGMA_MAX)) sigma = ALOE_II_SIGMA_MAX;
+const PROBE_SQRT_SCALER_MIN = 1.026248453011;
+const PROBE_SQRT_SCALER_MAX = 3.078745359035;
 
-  sigma = sigma.recklessMul(nSigma);
-
-  let a = sqrtMeanPriceX96.mul(ONE.sub(sigma).sqrt());
-  let b = sqrtMeanPriceX96.mul(ONE.add(sigma).sqrt());
-
-  if (a.lt(MIN_SQRT_RATIO)) a = MIN_SQRT_RATIO.recklessAdd(1);
-  if (b.gt(MAX_SQRT_RATIO)) b = MAX_SQRT_RATIO.recklessSub(1);
+function _computeProbePrices(sqrtMeanPriceX96: GN, iv: GN, nSigma: number): [GN, GN] {
```

Comment:
Why are we still using different logic than in earn? Shouldn't we make them the same?

Reply:
I know it's kinda annoying, but I didn't think it was worth the time right now.